docs/cheat-sheets/ruby-command-injection.mdx

```diff
@@ -17,11 +17,11 @@ import RecommendedRuleSet from "/src/components/code_snippets/_cheat-sheets-reco
 
```

Comment:
CommandInjectionIntro uses r2c instead of Semgrep

Reply:
I updated the branch from changes in the main branch, this issue is now fixed. :-)